roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -88,6 +101,7 @@
     "update_router:external_gateway_info:enable_snat": "rule:admin_only",
     "update_router:distributed": "rule:admin_only",
     "update_router:ha": "rule:admin_only",
+    "_COMMENT": "required for horizon",
```

Comment:
Is this for the new network tagging feature? I believe they let users tag their network through horizon.